README.md

```diff
@@ -11,3 +11,15 @@ This repository contains all the materials to be distributed through our worksho
 4. Add README.md containing all the resources and how to use that folder
```

Comment:
[A note on your avatar]

It seems that your `.gitconfig` is set up wrong. Please, check `user.name` and `user.email` matches your real name (Ganesh Tiwari) and GitHub email (ganesht049@gmail.com).

Reply:
alrighty i didnt think of that also i keep confusing my mail accounts.